src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -87,10 +86,10 @@ public T remove(int index) {
 
     @Override
     public boolean remove(T t) {
-        for(int i = 0; i < count; i++){
-            if(getNodeByIndex(i).item == t || getNodeByIndex(i).item.equals(t)){
-               remove(i);
-               return true;
+        for (int i = 0; i < count; i++) {
+            if (getNodeByIndex(i).item == t || getNodeByIndex(i).item.equals(t)) {
```

Comment:
Possible NPE here?

Reply:
forgot for NPE, try to make faster because dead line. Fixed.